components/schema-migrator/seeds/tenants/add_tenants.sql

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Why the first line is empty? And also move this file to `schema-migrator/seeds/director/` path